From 0f6857f2b5d9f53b96c22778f4d96cf3b3df49a4 Mon Sep 17 00:00:00 2001
From: Gilles Mazoyer <mazoyer.gilles@omega.ovh>
Date: Wed, 30 Sep 2015 16:06:45 +0200
Subject: [PATCH] Modification de la configuration par defaut du noyau pour la
 plateforme BCM63xx, pour que le modules spi-bcm63xx ne soit pas linke avec le
 noyau

---
 target/linux/brcm63xx/config-3.18 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/brcm63xx/config-3.18 b/target/linux/brcm63xx/config-3.18
index 0d7a5c7..0072637 100644
--- a/target/linux/brcm63xx/config-3.18
+++ b/target/linux/brcm63xx/config-3.18
@@ -194,7 +194,7 @@ CONFIG_RTL8367_PHY=y
 CONFIG_SERIAL_BCM63XX=y
 CONFIG_SERIAL_BCM63XX_CONSOLE=y
 CONFIG_SPI=y
-CONFIG_SPI_BCM63XX=y
+CONFIG_SPI_BCM63XX=n
 CONFIG_SPI_BCM63XX_HSSPI=y
 CONFIG_SPI_BITBANG=y
 CONFIG_SPI_GPIO=y
-- 
2.6.2

